earth_enterprise/src/maps/api/bootstrap.js

```diff
@@ -22,10 +22,20 @@ google.maps = google.maps || {};
     document.write('<script src="' + src +
                    '" type="text/javascript"></script>');
   }
+  
+  GEE_API_PATH = '/maps/api/js';
+  FUSION_API_PATH = '/maps/api';
 
   google.maps.Load = function(apiLoad) {
-    /* http://mt0.google.com/vt */    apiLoad([,[,,,],['en-US',,,,,,GEE_BASE_URL+'/maps/api/icons/',],[GEE_BASE_URL+'/maps/api/js','internal'],[0],0.0,,,,,], loadScriptTime);
+    /* http://mt0.google.com/vt */    apiLoad([,[,,,],['en-US',,,,,,GEE_BASE_URL + '/maps/api/icons/',],[GEE_BASE_URL + GEE_API_PATH,'internal'],[0],0.0,,,,,], loadScriptTime);
   };
+
   var loadScriptTime = (new Date).getTime();
-  getScript(GEE_BASE_URL+'/maps/api/js/main.js');
+
+  getScript(GEE_BASE_URL + GEE_API_PATH + '/main.js');
+  getScript(GEE_BASE_URL + GEE_API_PATH + '/../fusion_map_obj_v3.js');
+
+  getScript(GEE_BASE_URL + FUSION_API_PATH + '/fusion_extended_map.js');
```

Comment:
Minor: there's another file called `search_utils.js`, but perhaps it doesn't always need to be loaded, so it might be OK to leave it out here.